src/BetterPhpDocParser/PhpDocParser/DoctrineAnnotationDecorator.php

```diff
@@ -409,14 +409,20 @@ private function createDoctrineSpacelessPhpDocTagNode(
             $currentPhpNode
         );
 
+        $comment = $this->staticDoctrineAnnotationParser->getCommentFromRestOfAnnotation(
```

Comment:
Once we have processed the key/value pairs for the annotation we read the rest of the annotation as a comment